src/test/regress/multi_schedule

```diff
@@ -109,6 +109,7 @@ test: undistribute_table
 test: run_command_on_all_nodes
 test: background_task_queue_monitor
 test: other_databases grant_role_from_non_maindb
+test: seclabel_non_maindb
```

Comment:
If the following change doesn't increase the time it takes to execute those three tests in total, let's parallelize them:

```suggestion
test: other_databases grant_role_from_non_maindb seclabel_non_maindb
```